src/main/java/frc/robot/subsystems/Collector.java

```diff
@@ -58,9 +65,21 @@ public void setPivotMotorPosition(double position) {
     pivotMotor.set(ControlMode.Position, position);
   }
 
+  public Rotation2d getCollectorAbsoluteEncoder() {
```

Comment:
This can just return a double (the encoder counts). Arm was fancy so it needed the Rotation2d. this subsystem does not